contrib/freshmaker-cli

```diff
@@ -142,6 +153,19 @@ def get_api_url(deployment, env):
     return envs[env]
 
 
+def get_errata_filter(env):
+    """
+    Returns the address of custom precreated filter which shows the last 100 RHSA in SHIPPED_LIVE
+    state depending on the environment
+    """
+    ERRATA_FILTERS = {
+        "prod": "https://errata.engineering.redhat.com/filter/2964.json",
```

Comment:
When looking at the JSON, I don't see it as ordered by the date the advisory was shipped. How is it ordered?

The ordering must be correct in order for your code to find the advisories to work.